templates/zerver/app/index.html

```diff
@@ -125,7 +125,7 @@ <h3>{{ _('Loading...') }}</h3>
             {% include "zerver/app/right_sidebar.html" %}
         </div><!--/right sidebar-->
     </div><!--/row-->
-    <div class="informational-overlays overlay new-style" data-overlay="informationalOverlays" aria-hidden="true">
+    <div role="dialog" aria-labelledby="informationalTitle" class="informational-overlays overlay new-style" data-overlay="informationalOverlays" aria-hidden="true">
```

Comment:
Again `aria-labelledby` must always reference an existing `id`. Since this overlay doesn't contain a suitable title element, we can do `aria-label="{{ _('Help') }}"` instead of `aria-labelledby`.